.github/workflows/bump_version.yaml

```diff
@@ -0,0 +1,66 @@
+name: Bump Jobbergate Version
```

Comment:
Should we rename it to `Prepare for Release`? It would be more straightforward for the devs

Reply:
Yes, please.

Reply:
solved in [1077fbf](https://github.com/omnivector-solutions/jobbergate/pull/124/commits/1077fbfb52cada73757b00a695aca0fd98830741)